Source/PluginEditor.cpp

```diff
@@ -67,17 +69,29 @@ void SubsynthAudioProcessorEditor::paint (juce::Graphics& g)
     g.drawFittedText("Subsynth", 0, 0, getWidth(), 30, juce::Justification::centred, 1);
 }
 
-void SubsynthAudioProcessorEditor::resized()
-{
-    // This is generally where you'll want to lay out the positions of any
-    // subcomponents in your editor..
-    // 
+void SubsynthAudioProcessorEditor::resized() {
     // sets the position and size of the slider with arguments (x, y, width, height)
-    //freqSlide.setBounds(40, 30, 20, getHeight() - 60);
-    //freqLabel.setBounds(10, 10, 90, 20);
     waveSelect.setBounds(10, 20, 90, 20);
     keyboard.setBounds(10, 200, getWidth() - 20, getHeight() - 200);
     
     // ADSR Components
     adsrSliders.setBounds(150, 50, 400, 100);
+    
+    // gain slider
+    gainSlide.setBounds(700, 25, 100, 100);
+}
+
+// establish GUI configuration for gain rotary
+void SubsynthAudioProcessorEditor::setupGain() {
+    gainSlide.setSliderStyle(juce::Slider::Rotary);
+    gainSlide.setRotaryParameters(juce::MathConstants<float>::pi, (juce::MathConstants<float>::pi * 3), true);
+    gainSlide.setVelocityBasedMode(true);
+    gainSlide.onValueChange = [this] { gainSlide.setValue(gainSlide.getValue(), juce::dontSendNotification); };
+    gainSlide.setSkewFactor(2.0);
+    gainSlide.setRange(juce::Range<double>(-50.0, 0.0), 2.0);
+    gainSlide.setPopupDisplayEnabled(true, true, nullptr);
+    gainSlide.setValue(-25.0)
```

Comment:
semicolons are important apparently derp
```suggestion
    gainSlide.setValue(-25.0);
```